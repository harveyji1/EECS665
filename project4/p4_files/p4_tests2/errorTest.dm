
 v : () int{ 
    b : int;
    b : void;
}