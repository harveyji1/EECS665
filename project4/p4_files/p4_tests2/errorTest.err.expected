FATAL [4,5]-[4,6]: Invalid type in declaration
FATAL [4,5]-[4,6]: Multiply declared identifier
Name Analysis Failed